cmd/kogito/command/deploy/factory.go

```diff
@@ -21,6 +21,6 @@ import (
 
 // BuildCommands creates the commands available in this package
 func BuildCommands(ctx *context.CommandContext, rootCommand *cobra.Command) {
-	initDeleteServiceCommand(ctx, rootCommand)
 	initDeployCommand(ctx, rootCommand)
+	initDeleteServiceCommand(ctx, rootCommand)
```

Comment:
why moved ?

Reply:
Nothing intentional. I had made many changes around these files by mistake I forget to revert all changes on this file. Fixed it now